.github/workflows/deploy-demo-box.yml

```
@@ -170,50 +176,187 @@ jobs:
+  #
+  # psql on the box: still not installed, and there is still no passwordless
+  # sudo to install it. That is not worked around by wishing, it is worked
+  # around by running psql in a throwaway container on the stack's own network
+  # (scripts/stack-psql.sh), which the deploy job's price assertion uses too.
+  # bash, python3 and sha256sum, the other things scripts/apply-migrations.sh
+  # needs, are all present on the host already.
   # ------------------------------------------------------------------
   migrate:
     name: Apply pending database migrations
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, hive-demo]
```

actionlint (1.7.12)

[error] label "hive-demo" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)